### PR TITLE
Fix content/es/docs/setup/_index

### PR DESCRIPTION
**Pull request by jrab66**
Dangling "pick-right-solution" references #14474
es docs redirection  404 fixed

### CODE REVIEW

**Comment by jrab66** [comment]
I signed it

**Comment by raelga** [comment]
Hello @jrab66, thanks for taking care of that.

The original page has been re-written, so it may be better to change the entire page and apply the same rework done by @Rajakavitha1 at https://github.com/kubernetes/website/pull/14442. 

Otherwise, if you prefer to go ahead with this PR and focus on solving the broken links, I would suggest removing the entire link phrases: `Elige una [solución .......`. WIth the PR, they will be pointing to the top section of the same page, without any other extra content and is a bit misleading.

What do you think?

And thanks again for working on the issue! 🎉

**Comment by jrab66** [comment]
Hi @raelga  already remove the "Elige una solucion" stuff  its ok only doing this now and later aupdate with the translation of the full setup.

File: content/es/docs/setup/_index.md [hunk 1]
```diff
@@ -92,11 +87,9 @@ pero requiere mucho más conocimiento y experiencia. Este tipo de soluciones se
 pueden desplegar tanto en servidores físicos (_bare-metal_) como en instancias
 virtualizadas con diferentes sistemas operativos, en on-premises o en el cloud.
 
-Selecciona una [solución personalizada](/docs/setup/pick-right-solution/#custom-solutions).
 
 {{% /capture %}}
```
Comment (raelga) [inline content/es/docs/setup/_index.md]: Toda esta sección no es necesaria sin el enlace.

File: content/es/docs/setup/_index.md [hunk 2]
```diff
@@ -92,11 +87,9 @@ pero requiere mucho más conocimiento y experiencia. Este tipo de soluciones se
...
 
 {{% capture whatsnext %}}
```
Comment (raelga) [inline content/es/docs/setup/_index.md]:
```suggestion
```

File: content/es/docs/setup/_index.md [hunk 3]
```diff
@@ -92,11 +87,9 @@ pero requiere mucho más conocimiento y experiencia. Este tipo de soluciones se
...
 Para una lista completa de soluciones, consulta la sección
```
Comment (raelga) [inline content/es/docs/setup/_index.md]:
```suggestion
```

File: content/es/docs/setup/_index.md [hunk 4]
```diff
@@ -92,11 +87,9 @@ pero requiere mucho más conocimiento y experiencia. Este tipo de soluciones se
...
-[Escoge la solución correcta](/docs/setup/pick-right-solution/).
 {{% /capture %}}
```
Comment (raelga) [inline content/es/docs/setup/_index.md]:
```suggestion
```

File: content/es/docs/setup/_index.md [hunk 5]
```diff
@@ -83,7 +79,6 @@ Deberías elegir una solución de este tipo si:
 * Tienes un equipo de operaciones para desplegar y operar el clúster.
 * Tienes los recursos necesarios para ejecutar y monitorizar el clúster.
 
-Elige una [solución sobre virtualización On-Premises](/docs/setup/pick-right-solution/#on-premises-turnkey-cloud-solutions).
```
Comment (raelga) [inline content/es/docs/setup/_index.md]:
```suggestion
```

File: content/es/docs/setup/_index.md [hunk 6]
```diff
@@ -53,7 +52,6 @@ Deberías elegir una solución de este tipo si:
 * No tienes un equipo dedicado de operaciones pero quieres alta disponibilidad.
 * No tienes recursos para alojar y monitorizar sus clústeres.
 
-Elige una [solución gestionada](/docs/setup/pick-right-solution/#hosted-solutions).
```
Comment (raelga) [inline content/es/docs/setup/_index.md]:
```suggestion
```

File: content/es/docs/setup/_index.md [hunk 7]
```diff
@@ -38,7 +38,6 @@ Deberías elegir una solución de este tipo si buscas:
 * Probar o empezar a aprender sobre Kubernetes.
 * Desarrollar y testear clústeres localmente.
 
-Elige una [solución de máquina local](/docs/setup/pick-right-solution/#local-machine-solutions).
```
Comment (raelga) [inline content/es/docs/setup/_index.md]:
```suggestion
```

File: content/es/docs/setup/_index.md (hunk 1 above)
Comment (jrab66) [reply content/es/docs/setup/_index.md]: entonces elimino la seccion de ""## Soluciones personalizadas"" y los espacios extras.

File: content/es/docs/setup/_index.md [hunk 8]
```diff
@@ -83,20 +77,9 @@ Deberías elegir una solución de este tipo si:
...
-
 ## Soluciones personalizadas
 
 Una solución personalizadas proporciona total libertad sobre los clústeres
-pero requiere mucho más conocimiento y experiencia. Este tipo de soluciones se
-pueden desplegar tanto en servidores físicos (_bare-metal_) como en instancias
-virtualizadas con diferentes sistemas operativos, en on-premises o en el cloud.
-
-Selecciona una [solución personalizada](/docs/setup/pick-right-solution/#custom-solutions).
+pero requiere mucho más conocimiento y experiencia.
```
Comment (electrocucaracha) [inline content/es/docs/setup/_index.md]: `mucho mas` es como en los tapatios decimos `bien mucho`? :P Creo que con decir mas es suficiente
```suggestion
pero requiere más conocimiento y experiencia. 
```

**Comment by raelga** [comment]
Tracked at #13948.

**Comment by raelga** [comment]
Una vez solventada la review de @electrocucaracha, recuerda hacer un squash de los commits y dejarlo en uno solo, así podremos publicarlo directamente! :octocat: 

Muchas gracias @jrab66!

File: content/es/docs/setup/_index.md (hunk 8 above)
Comment (jrab66) [reply content/es/docs/setup/_index.md]: done :)

**Comment by raelga** [comment]
Muchas gracias!

Por mi parte:

/lgtm

A ver si @electrocucaracha puede confirmar que la review está resuelta y lo publicamos!

**Comment by raelga** [comment]
/approve

Muchas gracias!

**Comment by raelga** [comment]
Publicada en https://kubernetes.io/es/docs/setup/! 🎉

**Pull request merged**